tileview/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -772,7 +772,9 @@ public void onPanEnd( int x, int y, Origination origin ) {
 
   @Override
   public void onZoomBegin( float scale, Origination origin ) {
-    mDetailLevelManager.lockDetailLevel();
+    if( !Origination.DOUBLE_TAP.equals( origin ) ) {
```

Comment:
any zoom should not change detail levels while zooming.  it can change detail levels after, but not during - thus the lock here.  what's the rationale in doing this?